libchisel/src/depgraph.rs

```diff
@@ -1,22 +1,22 @@
 use std::collections::HashSet;
 
-use parity_wasm::elements::{CodeSection, Func, FuncBody, FunctionSection, Instruction, Module};
+use parity_wasm::elements::{FuncBody, Instruction, Module};
 
 /// A function dependency graph is represented as a list of "edges", or pairs of function indices
 /// (a, b) where a calls b.
 
 /// An edge, where the function at the left index calls the function at the right
 /// index.
 #[derive(PartialEq, Eq, Hash)]
-struct Edge(u32, u32);
+pub struct Edge(u32, u32);
 
 /// Container struct for the function dependency graph.
 pub struct DepGraph {
     edges: HashSet<Edge>,
 }
 
 /// Private interface for managing the function dependency graph
-trait DepGraphManager {
+pub trait DepGraphManager {
```

Comment:
In general I'm not happy that we're exposing these internal functions on the public trait.